internal/operators/manager_test.go

```diff
@@ -543,13 +543,20 @@ var _ = Describe("Operators manager", func() {
 		})
 
 		It("should be not valid if not enough disk space available for mce and odf", func() {
-			clusterHost = getMockHostWithDisks(int64(20), int64(20))
+			clusterHost = getMockHostWithDisks(int64(50), int64(50))
 
 			cluster.MonitoredOperators = []*models.MonitoredOperator{
 				&odf.Operator,
 				&mce.Operator,
 			}
 
+			// To get the right deployment mode
```

Comment:
Does `To get the right deployment mode` mean `target compact mode`?

Reply:
I think a comment like `// make sure it's compact mode` or something along those lines would be clearer